compiler/one-cmds/one-codegen

```diff
@@ -14,53 +19,93 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DRIVER_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-
-function Usage()
-{
-  echo "Usage: one-codegen [BACKEND] ..."
-  echo "Available BACKEND drivers:"
-  backend_exist=0
-  for file in `find $DRIVER_PATH -name *-compile -type f`;
-  do
-    backend_driver=$(basename $file)
-    sub_length=8
-    driver_length=$(expr ${#backend_driver} - ${sub_length})
-    backend=${backend_driver:0:${driver_length}} # 8 is length of "-compile"
-    echo "  $backend"
-    backend_exist=1
-  done
-  if [ $backend_exist == 0 ]; then
-    echo "  (There is no available backend drivers)"
-  fi
-
-  exit 255
-}
-
-function version()
-{
-  $DRIVER_PATH/one-version one-codegen
-  exit 255
-}
-
-# Get command from command-line
-BACKEND=$1
-if [[ -z ${BACKEND} ]]; then
-  Usage
-fi
-shift
-
-if [[ "${BACKEND}" == "--version" ]]; then
-  version
-fi
-
-BACKEND_DRIVER="${BACKEND}-compile"
-
-BACKEND_DRIVER_CMD="${DRIVER_PATH}/${BACKEND_DRIVER}"
-
-if [[ ! -f "${BACKEND_DRIVER_CMD}" ]]; then
-  echo "ERROR: '${BACKEND_DRIVER}' is not supported"
-  Usage
-fi
-
-"${BACKEND_DRIVER_CMD}" "$@"
+import argparse
+import os
+import subprocess
+import sys
+import tempfile
+
+import utils as _utils
+
+
+def _get_backends_list():
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    files = [f for f in os.listdir(dir_path) if os.path.isfile(os.path.join(dir_path, f))]
+    backends_list = []
+    for cand in files:
+        if cand.endswith('-compile'):
+            # 8 : length of '-compile'
+            backends_list.append(cand[:-8])
+    return backends_list
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(description='command line tool for code generation')
```

Comment:
`command` --> `Command` if there is no other message in front of this.
Please check with all other PRs.

Reply:
confirmed in another PR :)